source/features/filter-comments-by-you.tsx

```diff
@@ -1,21 +1,18 @@
-import React from 'dom-chef';
 import select from 'select-dom';
 import features from '../libs/features';
-import {getRepoURL} from '../libs/utils';
+import {getUsername, getRepoURL} from '../libs/utils';
 
 const repoUrl = getRepoURL();
 
 function init(): void {
-	select('.subnav-search-context .SelectMenu-list a:last-child')!
-		.before(
-			<a
-				href={`/${repoUrl}/issues?q=is%3Aopen+commenter:@me`}
-				className="SelectMenu-item"
-				role="menuitem"
-			>
-				Everything commented by you
-			</a>
-		);
+	const dropDownElement = select('#filters-select-menu a:last-child, .subnav-search-context li:last-child')!;
+	const newDropDownElement = dropDownElement.cloneNode(true);
+	const href = `/${repoUrl}/issues?q=is%3Aopen+commenter:${getUsername()}`;
+	const newDropDownElementLink = newDropDownElement.matches('a') ? newDropDownElement : select('a', newDropDownElement)!;
+	newDropDownElementLink.textContent = 'Everything commented by you';
+	newDropDownElementLink.setAttribute('href', href);
+	newDropDownElementLink.removeAttribute('target');
```

Comment:
Is it better to just copy the first link since it doesn’t have this attribute in the first place?

Reply:
and yes, we need to copy the first link, otherwise we just add the `is:open commenter:${getUsername()}` to a link targeting the external url `https://help.github.com/enterprise/2.18/user/articles/searching-issues`